### ISSUE TITLE
Question about Pretraining

### ISSUE THREAD

**Issue by necrophagists** [BlinkDL/RWKV-LM#298]
Hello! I wonder if RWKV7 used the sequence packing strategy during pre-training? If so, do the samples need to be masked from each other?

**Comment by BlinkDL** [comment]
Hi I am using simple packing (using [0] as eod) without inter-document mask
